practice.md

```diff
@@ -74,6 +74,25 @@ When an 18F project has a non-standard license status (e.g. it's a fork of a pre
 
 All 18F team members are expected to make new source code repositories public from the time of creation.
 
+### Managing 18F resources
+
+18F intends to produce great software for the American people. That means not just rushing through projects to get them working as fast as possible, but managing technical debt with an eye towards usability and reusability.
```

Comment:
I think "technical debt" here should be at the least hyperlinked or even defined in a very short parenthetical abstract.